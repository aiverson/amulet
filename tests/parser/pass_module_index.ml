let () =
  M.a
  M.(a + b)
  f M.{ a, b }
