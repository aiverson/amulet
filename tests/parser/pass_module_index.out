let () = begin
    M.a;
    M.(((a + b)));
    f M.({ a = a, b = b })
  end
